fs/info.py

```diff
@@ -71,6 +69,15 @@ def get(self, namespace, key, default=None):
         except KeyError:
             return default
 
+    def _require_namespace(self, namespace):
```

Comment:
Well, you could have decided that:
* if the namespace is available, use it
* if it's not, try to use the `type` namespace and check the `type`
* if `type` is not available either, then the FS is not supporting symlinks, so return False.

But I get your point, this is not a satisfying solution.